pinot-spi/src/main/java/org/apache/pinot/spi/config/table/FieldConfig.java

```diff
@@ -126,6 +126,9 @@ public enum CompressionCodec {
     SNAPPY(true, false),
     ZSTANDARD(true, false),
     LZ4(true, false),
+    // CLP is a special type of compression codec that isn't generally applicable to all RAW columns and has a
+    // special handling (see {@link CLPForwardIndexCreatorV1})
```

Comment:
> "a special handling"

It sounds like this is an incomplete sentence? Did you mean to say "special handling for log events" or something like that?

Reply:
Modified